content/docs/setup/kubernetes/install/cni/index.md

```diff
@@ -45,30 +45,73 @@ replaces the functionality provided by the `istio-init` container.
     1.  Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for any non-default
         settings required.
 
-1.  Add the `--set istio_cni.enabled=true` setting to the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/) to include the installation of the Istio CNI plugin. For example:
+1.  Update Helm's local package cache per the instructions in the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/).
+
+    1.  If installing via `helm template` fetch the `istio-cni` template in addition to the other Istio
```

Comment:
To help simplify things for readers, I suggest you add this content to the main step. I've left a suggestion that does this for you to be able to quickly implement it. I'm also removing the lines that then become redundant.
```suggestion
```